llm_api/llm_api.py

```diff
@@ -0,0 +1,25 @@
+import os
+
+api = os.environ.get('platform')
+
+if api == "openai":
+    from openai import openai
+    api = openai
+elif api == "claude":
+    from claude import claude
+    api = claude
+elif api == "qwen":
+    from qwen import qwen
+    api = qwen
+elif api == "qianfan":
+    from qianfan import qianfan
+    api = qianfan
```

Comment:
For this, maybe we could use [litellm](https://github.com/BerriAI/litellm)? It gives an interface to many models so we don't need to implement it ourselves.

Reply:
that saves a deal of work

Reply:
i add some from litellm. the rest don't have embedding, which is important for knowledge base